File: clients/r/test_client.R
source("fsda_client.R")

# Test 1: zscoreFS
data <- list(c(1,2,3), c(4,5,6), c(7,8,9))
result <- FSDA_call("zscoreFS", data)
print(result)

# Test 2: getYahoo
result2 <- FSDA_call("getYahoo", "AAPL", "plots", FALSE, "LastPeriod", "1mo")
print(result2$Ticker)
print(result2$Success)

File: clients/julia/test_client.jl
include("fsda_client.jl")

# Test 1: zscoreFS
data = [[1,2,3], [4,5,6], [7,8,9]]
result = fsda_call("zscoreFS", data)
println(result)

# Test 2: getYahoo
result2 = fsda_call("getYahoo", "AAPL", "plots", false, "LastPeriod", "1mo")
println(result2["Ticker"])
println(result2["Success"])